packages/osqp/meta.yaml

```
@@ -0,0 +1,21 @@
+package:
+  name: osqp
+  version: 1.0.0.b4
+  top-level:
+    - osqp
+requirements:
+  run:
+    - jinja2
```

Seems that the distribution name of Jinja2 has a capital J.
```suggestion
    - Jinja2
```